### ISSUE TITLE
When dragging HTML files onto Yimello, sometimes they are displayed within

### ISSUE THREAD

**Issue by ghost** [acadet/yimello#6]
Hello there.

Just to expand: this happens only when the 'copy' cursor is displayed (cursor with plus sign). It also removes the navigation bar at the top. Essentially, it turns Yimello into a web browser.

Thank you!

**Comment by acadet** [comment]
Yes, it makes sense. It is caused by node-webkit tool. Actually, Node-webkit is an emulator, based on webkit engine, that's why, if you drag an html page, it is trying to run it.

I will figure this issue out as soon as possible.

**Comment by Aedx** [comment]
It isn't really an emulator, rather a version of Chromium which runs off of node.js - but there should be some way of preventing users being able to access most of it (aka other web pages).
